api/proto/teleport/devicetrust/v1/usage.proto

```diff
@@ -18,27 +18,16 @@ package teleport.devicetrust.v1;
 
 option go_package = "github.com/gravitational/teleport/api/gen/proto/go/teleport/devicetrust/v1;devicetrustv1";
 
-// AccountUsageType is the type of the underlying account, either limitless or
-// limit-based.
+// Superseded by ResourceUsageService.GetUsage.
 enum AccountUsageType {
   ACCOUNT_USAGE_TYPE_UNSPECIFIED = 0;
   ACCOUNT_USAGE_TYPE_UNLIMITED = 1;
   ACCOUNT_USAGE_TYPE_USAGE_BASED = 2;
 }
 
-// DevicesUsage holds aggregated information about trusted device usage.
+// Superseded by ResourceUsageService.GetUsage.
 message DevicesUsage {
-  // Usage type of the underlying account.
-  // UNLIMITED accounts have no limits on the number of trusted devices,
-  // therefore all usage-based limits are data are zeroed when read.
```

Comment:
Why delete the documentation comments?

Reply:
The idea is that the "superseded by" comment is all that matters, everything else is likely to become obsolete over time. I might delete the message contents in a follow up too.